### PR TITLE
OpenTelemetry spans for gRPC ext_authz calls

### PR DESCRIPTION
**Pull request by vinhph0906**
I re-create PR #371 to add tracing to OPA-Envoy-Plugin.

### CODE REVIEW

**Comment by LavaToaster** [comment]
Anything I can do to help with this?

cc @ashutosh-narkar @srenatus

**Comment by srenatus** [comment]
@LavaToaster thanks for the poke. The e2e tests have been fixed in the mean time, so I've updated this branch. If it goes green, I'll merge it.

**Review by srenatus — APPROVED** [review]
Thanks for getting back to this 👏

File: test/e2e/testing.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	}`, addr, query)
+	cfg, err := internal.Validate(m, []byte(cfgJSON))
+	if err != nil {
+		return nil, err
+	}
+	m.Register(plugin.PluginName, internal.New(m, cfg))
+	if err := m.Start(ctx); err != nil {
+		return nil, err
+	}
+	return m, nil
```
[the remark this answers is not in the document]
Comment (srenatus) [reply test/e2e/testing.go:43]: Doesn't matter.

**Pull request merged**